fixtures.js

```diff
@@ -0,0 +1,32 @@
+//This file is needed to use the tests with the extension installed and running
+import { test as base, chromium } from '@playwright/test';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export const test = base.extend({
+  // eslint-disable-next-line no-empty-pattern
+  context: async ({}, use) => {
+    const pathToExtension = path.join(__dirname, 'dist/chrome');
+    const context = await chromium.launchPersistentContext('', {
+      headless: false,
+      // To run and debug the extension in head mode remove `--headless=new` from args
```

Comment:
This comment is not relevant anymore. Consider commenting the research you did, such as "using --headless=new didn't work on Github Actions"

Reply:
the github actions needs to be more researched I truly don't understand whats wrong there, tried few things but it didn't work.
if you add the `--headless=new` to the args and run you will see no browser is opened but once you remove it a new window will open.
if it doesn't work we can do  a zoom meeting

Reply:
Just take a look here: https://github.com/brafdlog/caspion/blob/72b01afb3a7e46cd4b8906885a2c73d56b5174ec/.github/workflows/ci.yml#L39-L45
I don't think we are solving there the exact same issue, but it is close enough, we are running an Electron app with Cypress.

Reply:
I think I solved the issue, the pipeline should run now.